composer.json

```diff
@@ -39,16 +39,16 @@
         "fig/http-message-util": "^1.1.2",
         "laminas/laminas-stdlib": "^3.19.0",
         "mezzio/mezzio-router": "^3.18 || ^4.0.1",
-        "nikic/fast-route": "^1.2",
+        "nikic/fast-route": "^2.0.0-beta1",
```

Comment:
From my PoV, this is OK, but we need to wait for a stable release before releasing on our end

Reply:
Version 1 works just fine - I made this patch more so that I could test the beta and provide some feedback to @lcobucci eg https://github.com/nikic/FastRoute/pull/302 - There's no rush!